spec/features/courses/preview_spec.rb

```diff
@@ -146,10 +146,6 @@
       "£9,250",
     )
 
-    expect(preview_course_page.eu_fees).to have_content(
```

Comment:
I think it might be worth removing `eu_fees` element from the SitePrism object.

Reply:
done!